core/io/marshalls.cpp

```diff
@@ -379,8 +379,7 @@ Error decode_variant(Variant &r_variant, const uint8_t *p_buffer, int p_len, int
 		} break;
 		/*case Variant::RESOURCE: {
 
-			ERR_EXPLAIN("Can't marshallize resources");
-			ERR_FAIL_V(ERR_INVALID_DATA); //no, i'm sorry, no go
+			ERR_FAIL_V_MSG(ERR_INVALID_DATA, "Can't marshallize resources"); //no, i'm sorry, no go
```

Comment:
That's actually dead code, there hasn't been `Variant::RESOURCE` in years, so you could remove the commented out code.